IdentityCore/src/webview/response/MSIDWebWPJResponse.m

```diff
@@ -70,6 +67,33 @@ - (instancetype)initWithURL:(NSURL *)url
     return self;
 }
 
+- (BOOL)isBrokerInstallResponse:(NSURL *)url
+{
+    NSString *scheme = url.scheme;
+    NSString *host = url.host;
+    
+    if ([scheme isEqualToString:@"msauth"] && [host isEqualToString:@"wpj"])
+    {
+        return YES;
+    }
+    
+    NSArray *pathComponents = url.pathComponents;
+    
+    if ([pathComponents count] < 2)
+    {
+        return NO;
+    }
+    
+    NSUInteger pathComponentCount = pathComponents.count;
```

Comment:
It will be better if we add some comments for possible install link format, that will definitely ease the pain of people reading this code :)